dotCMS/src/main/java/com/liferay/portal/action/LogoutAction.java

```diff
@@ -66,7 +66,7 @@ public ActionForward execute(ActionMapping mapping, ActionForm form, HttpServlet
 			}
 
 			try {
-				this.loginService.doActionLogout(req, res);
+				this.loginService.doLogout(req, res);
 			} catch (Exception e) {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtypes instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2221)